packages/browser/jest.setup.js

```diff
@@ -2,7 +2,13 @@
 // https://github.com/jsdom/jsdom/issues/1612
 /* eslint-disable @silverhand/fp/no-mutation */
 /* eslint-disable unicorn/prefer-module */
+const { Crypto } = require('@peculiar/webcrypto');
+const { location } = require('jest-location-mock');
+const fetch = require('node-fetch');
 const { TextDecoder, TextEncoder } = require('text-encoder');
 
+global.crypto = new Crypto();
+global.location = location;
+global.fetch = fetch;
```

Comment:
We do we need fetch in unit test?

Reply:
createRequester/Requester uses fetch.

Reply:
Removed node-fecth and improved UT. @gao-sun PTAL.

Reply:
Improved. @gao-sun PTAL.
```ts
const crypto = require('crypto');

……

global.crypto = {
  getRandomValues: (buffer) => crypto.randomFillSync(buffer),
  subtle: crypto.webcrypto.subtle,
};
```